iosApp/iosApp/background/IosBackgroundRunner.swift

```diff
@@ -0,0 +1,70 @@
+import composeApp
+import AVFAudio
+import UIKit
+
+class IosBackgroundRunner : BackgroundRunner {
+    private var backgroundTask: UIBackgroundTaskIdentifier = .invalid
+    private var audioPlayer: AVAudioPlayer?
+
+    /// Invoke the given task in the background.
+    func invoke(background: @escaping ()  -> Void) {
+        let serialQueue = DispatchQueue(label: "org.openobservatory.queue", qos: .userInitiated, attributes: [.concurrent])
```

Comment:
Shouldn't the label differ from depending on the applicationID or package? Or there's no possibility of conflict between apps?

Reply:
ah. good point. Will correct this.